lib/commands/apps-management.js

```diff
@@ -0,0 +1,94 @@
+import _ from 'lodash';
+import waitForCondition from 'asyncbox';
+import {fs, utils} from 'appium-support';
+import log from '../logger';
+
+let commands = {};
+const APP_STATE_NOT_INSTALLED = 0;
```

Comment:
are we making up these constants or do they map to something in android land?

Reply:
I just wanted to make them similar the corresponding iOS constants.